lib/open_api_spex/cast/error.ex

```diff
@@ -71,10 +228,67 @@ defmodule OpenApiSpex.Cast.Error do
     end
   end
 
+  def message(%{reason: :all_of, value: value, meta: %{invalid_schema: invalid_schema}}) do
+    "Failed to cast #{inspect(value)} as #{invalid_schema}. Value must be castable using `allOf` schemas listed."
+  end
+
+  def message(%{reason: :any_of, value: value, meta: %{failed_schemas: failed_schemas}}) do
+    "Failed to cast #{inspect(value)} using any of: #{failed_schemas}"
+  end
+
+  def message(%{
+        reason: :one_of,
+        value: value,
+        length: 0,
+        meta: %{failed_schemas: failed_schemas}
+      }) do
+    "Failed to cast #{inspect(value)}. Value must be castable using only `oneOf` the schemas provided, found: #{
```

Comment:
Here too, I've been avoiding dumping the given value in the error message, because it presents a potential DOS attack vector. What do you think?